apps/docs/src/content/docs/reference/sheepdog-utils.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Sheepdog Utils
+description: the utilities object that is available in every task
+---
+
+When creating a task (of any type) there are 2 possible arguments that be accessed, the first is any argument that is passed in when calling `.perform()`, and the second is an object that can be used to extend your task - `SheepdogUtils`.
```

Comment:
I'm not convinced of the word "extend" here...it's more like, stuff that you might need inside your task. But i don't know how to express it less weirdly.

Reply:
What about "enhance"?